ParcelInvoiceMaker/ParcelInvoiceMaker/Models/Name.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Name.swift
+//  ParcelInvoiceMaker
+//
+//  Created by JunHeeJo on 1/26/24.
+//
+
+import Foundation
+
+struct Name {
+    let value: String
+    
+    init(_ value: String) throws {
+        let koreanNameRegex: String = "^[\(KoreanNameRules.firstKoreanUnicode)-\(KoreanNameRules.lastKoreanUnicode)]{\(KoreanNameRules.minNameCount),\(KoreanNameRules.maxNameCount)}$"
+        guard  value.range(of: koreanNameRegex, options: .regularExpression) != nil else { throw NSError() }
```

Comment:
```suggestion
        guard value.range(of: koreanNameRegex, options: .regularExpression) != nil else { throw NSError() }
```

p5;

띄어쓰기가 2개 들어간 것으로 보여요 👀